index.d.ts

```diff
@@ -50,3 +50,35 @@ export interface Statement {
 export interface StatementMap {
 	[ propertyId: string ]: Statement[];
 }
+
+export interface Term {
+	language: string;
+	value: string;
```

Comment:
This could also include the optional `for-language` key which `wbgetentities` can return when language fallbacks are enabled ([example URL](https://www.wikidata.org/w/api.php?action=wbgetentities&ids=Q12345&props=labels&languages=de-formal&languagefallback=1)), but if that’s not used in Termbox then it’s not required :)

Reply:
`for-language` is not used in termbox, no.